packit.spec

```diff
@@ -0,0 +1,58 @@
+# Created by pyp2rpm-3.3.2
+%global pypi_name we-dont-know-yet
+%global real_name packit
+
+Name:           %{real_name}
+Version:        0.1.0
+Release:        1%{?dist}
+Summary:        A set of tools to integrate upstream open source projects into Fedora operating system
+
+License:        MIT
+URL:            https://github.com/packit-service/packit
+Source0:        TBD
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  rebase-helper
+BuildRequires:  python3dist(setuptools-scm)
+BuildRequires:  python3dist(setuptools-scm-git-archive)
+
```

Comment:
We should require a subpackage here, or not?

(`Requires python3-%{real_name}`)

Reply:
+1